styles.css

```diff
@@ -819,3 +822,192 @@ div.item26 {
   align-self: flex-end;
   opacity: 0.9;
 }
+
+.container-contact-me {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 320px;
+  height: 813px;
+  margin-top: 5vh;
+  background: #f9f9fa;
+  background-image: url(./images/img1.svg);
+  background-repeat: no-repeat;
+}
+
+.get-started {
+  margin-right: 75px;
+  margin-top: 106px;
+  height: auto;
+}
+
+.second-text {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  color: #1a2236;
+  height: 63px;
+  width: 288px;
+  margin-left: 16px;
+}
+
+.input-label {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 288px;
+  height: 52px;
+  margin-bottom: 20px;
+  margin-left: 9px;
+  background: #fff;
+  border: 1px solid #dfdfe0;
+  border-radius: 4px;
+}
+
+.label {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 15px;
+  height: 15px;
+  width: auto;
+  margin-top: 2px;
+  margin-left: 16px;
+  margin-bottom: 4px;
+  color: #7d7c84;
+}
+
+.input {
+  border: 0;
+  outline: 0;
+  margin-left: 16px;
+  width: 260px;
+  height: 84px;
+}
+
+.input:focus {
+  outline: none !important;
+}
+
+#message-label {
+  font-size: 14px;
+  line-height: 21px;
+}
+
+#message {
+  font-family: 'Poppins', sans-serif;
+  font-size: 14px;
+  line-height: 21px;
+}
+
+#message-area {
+  width: 288px;
+  height: 112px;
+}
+
+.submit-button {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 288px;
+  height: 44px;
+  margin-left: 9px;
+  background: #1a2236;
+  margin-bottom: 50px;
+}
```

Comment:
[OPTIONAL] When hovering over a button (or link), the cursor shape should change to be a pointer. I suggest you add `cursor: pointer;` to make that happen, preferably to the button selector.